backend/src/resolvers/resolvers.ts

```diff
@@ -0,0 +1,19 @@
+import { MutationResolvers, QueryResolvers, Resolvers } from "../generated/graphqlgen";
+import { PostMutationResolver, PostQueryResolver } from "./post.resolver";
+import { UserQueryResolver } from "./user.resolver";
+import { NestedFields } from "./nested/nestedFields";
+
+const Query: QueryResolvers.Type = {
+    ...PostQueryResolver,
+    ...UserQueryResolver
+};
+
+const Mutation: MutationResolvers.Type = {
+    ...PostMutationResolver
+};
+
+export const resolvers: Resolvers = {
+    Query,
+    Mutation,
+    ...NestedFields
+};
```

Comment:
this one file would maybe also be enough to implement the resolvers. (with less code in the end)